### PR TITLE
[CPM-184] programme regulation scrolling

### PR DESCRIPTION
**Pull request by Richardng12**
## Changes

- Fixed scrolling bug with programme requirements, no longer need to hold shift down to scroll horizontally.

## Checklist

- [x] Have tests been written?
- [x] Is the UI consistent?

## Requests
- [ ] Can somebody check the styling/CSS?

Addresses [CPM-184]

Connects #y

[CPM-184]: https://momentum-agile.atlassian.net/browse/CPM-184

### CODE REVIEW

File: seed-db/scripts/plan.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ const createPlan = (i, program, student) => {
         numYears: getRandomNumberInRange(1, 3),
         completed: false,
         createdAt: new Date().toISOString(),
-        updatedAT: new Date().toISOString(),
+        updatedAt: new Date().toISOString(),
```
Comment (daltonlim) [inline seed-db/scripts/plan.js:47]: Does this field show up anywhere on the frontend?

File: client/src/pages/Programmes/scrollHook.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { useRef, useEffect } from "react";
+
+export const useHorizontalScroll = () => {
+  const elRef = useRef();
+  useEffect(() => {
```
Comment (daltonlim) [inline client/src/pages/Programmes/scrollHook.js:5]: image
Are you able to make this part scroll

File: seed-db/scripts/plan.js (hunk 1 above)
Comment (Richardng12) [reply seed-db/scripts/plan.js:47]: I think it does for something, makes the objetct stuff weird, i just accidentally named it wrong at the beginning

File: client/src/pages/Programmes/scrollHook.js (hunk 2 above)
Comment (Richardng12) [reply client/src/pages/Programmes/scrollHook.js:5]: That part scrolls for me on my pc

Comment (daltonlim) [reply client/src/pages/Programmes/scrollHook.js:5]: hmm, it only fails in editing mode but the other ones work

Comment (vee-christella) [reply client/src/pages/Programmes/scrollHook.js:5]: It fails for me too, even when I scroll horizontally with my trackpad

Comment (Richardng12) [reply client/src/pages/Programmes/scrollHook.js:5]: Fixed ref reference, can you guys try again

**Comment by daltonlim** [comment]
@pull-dog up

**Review by daltonlim — APPROVED** [review]
LGTM

**Review by FocalChord — APPROVED** [review]
L G T M

**Pull request merged**